### DISCUSSION TITLE
How to change color of an specific candle?

### DISCUSSION THREAD

**Discussion by cleyversoncosta** [tradingview/lightweight-charts#719] [General] [answered]
How to change color of an specific candle?
1) after being plot
2) on real time

the code below changes the whole series and I would like to change color of an specific candle

candleSeries.applyOptions({
                    upColor: 'rgba(255, 0, 0, 1)',
                    downColor: 'rgba(0, 255, 0, 1)',
                });

**Comment by timocov** [comment, marked as answer]
It's impossible to do so. See https://github.com/tradingview/lightweight-charts/issues/195. But even if and when https://github.com/tradingview/lightweight-charts/issues/195 will be implemented, you have to call `setData` to update a candle/bar color if that bar is not last one.

**Reply by roysupriyo10** [reply]
visit the examples section in the documentation, I think they have a tutorial where they show how to change colors of specific candles

**Reply by Jiamingou** [reply]
https://tradingview.github.io/lightweight-charts/tutorials/customization/data-points
